### DISCUSSION TITLE
Is there a toggle for clear vision?

### DISCUSSION THREAD

**Discussion by ivjvon** [winstxnhdw/lc-hax#285] [Q&A]
If so whats the button and can you rebind the toggles?

**Comment by winstxnhdw** [comment]
There is none. All available toggles are documented in the README. Theoretically, you can rebind the toggles from [here](https://github.com/winstxnhdw/lc-hax/blob/main/lc-hax/Scripts/Core/InputListener.cs), but if you know nothing about programming, you might struggle a bit.

**Reply by ivjvon** [reply]
Yep I know nothing. Is it possible if you can make a .config for it?

**Reply by winstxnhdw** [reply]
I could, but I would have to rewrite the entire input module. We also make use of more complex key combinations like key holds which will make implementing this a bit more complicated. I would also have to write some conflict resolution logic at runtime. I wouldn’t expect this any time soon.

Of course if you think certain keys are better placed elsewhere and it makes sense to everyone, I am more than happy to change it.

**Reply by ivjvon** [reply]
I would like the majority to vote for this but this feature is pretty useful
